shell/platform/darwin/ios/ios_surface_software.mm

```diff
@@ -118,7 +120,7 @@
     return false;
   }
 
-  layer_.get().contents = reinterpret_cast<id>(static_cast<CGImageRef>(pixmap_image));
+  layer_.get().contents = (__bridge id)(static_cast<CGImageRef>(pixmap_image));
```

Comment:
Couldn't we remove the scoped_nsobject from `layer_`?

But it's also fine to defer that to a second-pass cleanup when you eliminate `scoped_nsobject` completely if you'd rather.

Reply:
Let's do a cleanup sweep at the end.  I'll keep removing them where it's easy though.